doc/iterations/2024.1/shapes/add_support_for_libraries.adoc

```diff
@@ -0,0 +1,39 @@
+= Add the support for libraries
+
+== Context
+
+Users want to have access to read only data (a.k.a libraries).
+
+== Key Result
+
+In the Details View, users could reference these read only data from their models' elements.
+
+== Solution
+
+1. When declaring templates, specifiers should be able to specify a list of libraries that will be accessible in the projects created from the templates.
+
+2. Specifiers should be able to associate libraries to projects' natures. Thus, when a project with a specific nature is created/uploaded, the libraries associated to this specific nature will be accessible.
```

Comment:
Why such coupling? Two projects with the same nature may have a different set of libraries (see Papyrus Web).

Reply:
done